frontend/assets/l10n/app_en.json

```diff
@@ -99,7 +99,7 @@
     "codeExpired": "The scanned code has already expired on {{expirationDate}}.",
     "codeInvalid": "The content of the scanned code cannot be understood. It is probably a QR code that was not generated for this app.",
     "codeInvalidMissing": "The content of the scanned code is incomplete. (Error code: {{missing}}Missing)",
-    "codeInvalidType": "The scanned code is not valid for this operation. (Scanned: {{qrCodeType}})",
+    "codeInvalidType": "The scanned code is not valid for activation. (Scanned: {{qrCodeType}}). Please use the QR code shown as activation code.",
```

Comment:
just a random thought during testing: is it helpful to show qrCodeType to user?
for example, I see `Scanned: static verification code`, but do users know what it means?

Reply:
Okay we might map this to sth more intuitive

Reply:
@seluianova i removed this part with "what was scanned" since i think this is not really import to the user. Please check the adjusted message, if thats okay for you

Reply:
@f1sh1918 looks good!